daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Error.scala

```diff
@@ -68,16 +69,24 @@ object Error {
 
   object Preprocessing {
 
-    sealed abstract class Error extends RuntimeException with scala.util.control.NoStackTrace {
+    sealed abstract class Error
+        extends RuntimeException
```

Comment:
do we ever throw this?

Reply:
We throw `Preprocessing.ValueNesting`,  `Error.Preprocessing.TypeMismatch`, `Error.Preprocessing.RootNode`